hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/PrefetchExecutor.java

```diff
@@ -53,7 +56,8 @@ public final class PrefetchExecutor {
   private static final Map<Path, Future<?>> prefetchFutures = new ConcurrentSkipListMap<>();
   /** Set of files for which prefetch is completed */
   @edu.umd.cs.findbugs.annotations.SuppressWarnings(value = "MS_SHOULD_BE_FINAL")
-  private static HashMap<String, Boolean> prefetchCompleted = new HashMap<>();
+  private static ConcurrentHashMap<String, Long> regionPrefetchSizeMap = new ConcurrentHashMap<>();
+  private static HashMap<String, Map<String, Long>> prefetchCompleted = new HashMap<>();
```

Comment:
Should use `Map<String,Pair<String,Long>> prefetchCompleted`.

Reply:
Updated the code as per the suggestion. Replaced HashMap and ConcurrentHashMap to Map.

Reply:
How about using Pair<String,Long> as the map value? We don't really need a map as the value here. You can use `org.apache.hadoop.hbase.util.Pair` here, I believe it would be clearer to understand the structure.

Reply:
Good catch Wellington, Thanks for pointing it out. Made the change in the updated patch.